Sources/LanguageServerProtocol/SupportTypes/ServerCapabilities.swift

```diff
@@ -194,16 +194,21 @@ public struct TextDocumentSyncOptions: Codable, Hashable {
     public init(includeText: Bool = false) {
       self.includeText = includeText
     }
+
+    public init(from decoder: Decoder) throws {
+      let container = try decoder.container(keyedBy: CodingKeys.self)
+      self.includeText = try container.decodeIfPresent(Bool.self, forKey: .includeText) ?? false
```

Comment:
Shouldn't `includeText` just be optional?

Reply:
I could swap to that, but they're functionality equivalent, this just makes it easier to handle since the default is false

Reply:
Originally that is how I was implementing these structs too: making them more convenient, but I now lean toward making it possible to round-trip values without changing them.  This is valuable both for our interaction with clangd (we don't want to silently modify the results), and if anyone wants to use our LSP layer as a library for other languages in the future.

Reply:
Ah I see, swapped over to optional